ansible/.env.j2

```diff
@@ -0,0 +1,12 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+FLASK_ENV=production
+
+TRELLO_API_KEY={{ trello_api_key }}
+TRELLO_API_TOKEN={{ trello_api_token }}
+TRELLO_BOARD_ID={{ trello_board_id }}
+
+TRELLO_TODO_LIST_ID={{ trello_todo_list_id }}
+TRELLO_DONE_LIST_ID={{ trello_done_list_id }}
```

Comment:
Worth noting these wouldn't _need_ to be encrypted, since they're not sensitive strictly speaking, but certainly no harm in them being so, and having them pulled out as variables is definitely a positive 😄